### ISSUE TITLE
Controlnet

### ISSUE THREAD

**Issue by Tobe2d** [Sanster/lama-cleaner#322]
I updated to the latest release 1.2.0 but I cant find the Controlnet options...

More over, I activated it in settings under stable diffusion but still cant find it ;-(

What am I missing?

![image](https://github.com/Sanster/lama-cleaner/assets/4099839/52c5d674-2c4e-4fb7-8ec8-3aef68b5ffcb)

![image](https://github.com/Sanster/lama-cleaner/assets/4099839/77e8f511-7174-4dde-92a0-1de15c706d5a)

**Comment by Sanster** [comment]
ControlNet is not an independent model, so it does not appear in the settings. You can use the following options to start the service:

```bash
--model sd1.5 --sd-controlnet --sd-controlnet-method control_v11p_sd15_canny
```

**Comment by Tobe2d** [comment]
Thanks for the reply @Sanster 
I really dont know where to add this argument, I am using the one click installer, is that why?

**Comment by Tobe2d** [comment]
One more question please, when I am using the latest update I am not geting alphachannel from remove background.
so it is saving .png but with bkacj background not transparant any idea why?

Please help me on fixing this for alpha channel in remove BG and where to use `--model sd1.5 --sd-controlnet --sd-controlnet-method control_v11p_sd15_canny`

**Comment by Sanster** [comment]
## ControlNet

1. Set Stable Diffusion model(Currently there is a bug where if the initial model launched is not an SD model, ControlNet will not take effect.) as the initial launched model: sd1.5/anything4/realisticVision1.4

<img width="786" alt="截屏2023-06-10 21 00 40" src="https://github.com/Sanster/lama-cleaner/assets/3998421/fadffd93-587d-4db0-809a-2220b38a9977">

2. Enable ControlNet in the Diffusion Model tab

<img width="942" alt="截屏2023-06-10 21 03 55" src="https://github.com/Sanster/lama-cleaner/assets/3998421/8a11ce01-55a2-4fc5-8357-a482b5a4b0b2">

## Remove background issue
I did not reproduce this issue. My PNG images have transparent background layers after being saved.

my test image:

![dog](https://github.com/Sanster/lama-cleaner/assets/3998421/75420fda-5f7a-4338-8932-3cdc78ee4380)

remove bg resule:

![dog_cleanup](https://github.com/Sanster/lama-cleaner/assets/3998421/e115e8c4-df1d-4d07-95bd-c497a92cfd3a)

**Comment by Tobe2d** [comment]
@Sanster Thanks a lot, once I set sd1.5 it work as expected.

Regarding removeBG I test your image and below is the output:

![244872630-75420fda-5f7a-4338-8932-3cdc78ee4380](https://github.com/Sanster/lama-cleaner/assets/4099839/ed06f853-d356-46f6-91f6-aef090e2befd)

**Comment by Sanster** [comment]
rembg alpha channel issue fixed in 1.2.1

**Comment by Tobe2d** [comment]
Yes I can confirm it is working now... thank you!
